scripts/apps/templates/directives/TemplateSelectDirective.ts

```diff
@@ -43,10 +43,10 @@ export function TemplateSelectDirective(api, desks, session, templates, notify)
                             scope.publicTemplates = [];
                             scope.privateTemplates = [];
                             result._items.forEach((template) => {
-                                if (template.is_public !== false) {
-                                    scope.publicTemplates.push(template);
-                                } else {
+                                if (template.is_public === false) {
                                     scope.privateTemplates.push(template);
+                                } else if (template.template_desks && template.template_desks.length > 0) {
```

Comment:
What is this condition for?

Reply:
If the template is not assigned to any desk, then it shouldn't be shown when creating a new item.
That's what the ticket is all about.

Reply:
Got it, thanks. I didn't notice you were already getting the templates that are only accessible to the desk & user.